### PR TITLE
docs(tabs): make tabs content(tab-pane) looks nice #3642

### LINKED ISSUE #3642: docs(tabs): make tabs content(tab-pane) looks nice
Make tabs content(tab-pane) looks nice (related to #3267)

### PR DESCRIPTION
**Pull request by IraErshova**
Fill tabs demo with styled content

Close #3642

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

File: demo/src/app/components/+tabs/demos/basic/basic.ts [hunk 1]
```diff
@@ -2,12 +2,13 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'demo-tabs-basic',
-  templateUrl: './basic.html'
+  templateUrl: './basic.html',
+  styles: ['.tab-container {padding: 10px 10px 0;}']
```
Comment (valorkin) [inline demo/src/app/components/+tabs/demos/basic/basic.ts]: why do we need custom styles? 
is it intended part of demo?

File: demo/src/app/components/+tabs/demos/config/config.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ export function getTabsetConfig(): TabsetConfig {
 @Component({
   selector: 'demo-tabs-config',
   templateUrl: './config.html',
+  styles: ['.tab-container {padding: 10px 10px 0;}'],
```
Comment (valorkin) [inline demo/src/app/components/+tabs/demos/config/config.ts]: try to avoid custom styles

**Review by valorkin — CHANGES_REQUESTED** [review]
please remove `styles` usage in demos if not needed
by demo goal

**Review by IlyaSurmay — DISMISSED** [review]

**Review by valorkin — APPROVED** [review]

**Comment by valorkin** [comment]
it will be easier to redo than to resolve conflicts :)

**Pull request closed without merging**